include/net/nrf_cloud.rst

```diff
@@ -22,64 +25,44 @@ Before using any other APIs of the module, the application must call :cpp:func:`
 
 Connecting
 **********
-The application can use :cpp:func:`nrf_cloud_connect` to connect to the cloud. This API triggers a series of events and actions in the system. If the API fails, the application must retry to connect.
-:cpp:func:`nrf_cloud_connect` requires a list of supported features in the application.
-Use the following code in your application to initialize this list:
+The application can use :cpp:func:`nrf_cloud_connect` to connect to the cloud.
+This API triggers a series of events and actions in the system.
+If the API fails, the application must retry to connect.
 
-
-.. code-block:: c
-
-   /* Array of supported user association methods. */
-   const enum nrf_cloud_ua supported_uas[] = {
-		NRF_CLOUD_UA_BUTTON
-   };
-
-   /* Array of supported sensors. */
-   const enum nrf_cloud_sensor supported_sensors[] = {
-		NRF_CLOUD_SENSOR_GPS,
-		NRF_CLOUD_SENSOR_FLIP
-   };
-
-   /* List of supported user association methods. */
-   const struct nrf_cloud_ua_list ua_list = {
-		.size = ARRAY_SIZE(supported_uas),
-		.ptr = supported_uas
-   };
-
-   /* List of supported sensors. */
-   const struct nrf_cloud_sensor_list sensor_list = {
-		.size = ARRAY_SIZE(supported_sensors),
-		.ptr = supported_sensors
-   };
-
-   /* Struct containing both lists. */
-   const struct nrf_cloud_connect_param param = {
-		.ua = &ua_list,
-		.sensor = &sensor_list,
-   };
-
-   int err = nrf_cloud_connect(&param);
-
-
-First, the library tries to establish the transport for communicating with the cloud. This procedure involves a TLS handshake that might take up to three seconds. The API blocks for the duration of the handshake.
+First, the library tries to establish the transport for communicating with the cloud.
+This procedure involves a TLS handshake that might take up to three seconds.
+The API blocks for the duration of the handshake.
 
 Next, the API subscribes to an MQTT topic to start receiving user association requests from the cloud.
+The cloud uses the device's certificates for authentication.
+The certificates are generated using the device's ID and PIN/HWID.
 
-Every time nRF Cloud starts a communication session with a device, it verifies if the device is uniquely associated with a user. If not, the user association procedure is triggered. The following message sequence chart shows the flow of events and expected application responses to each event:
+Every time nRF Cloud starts a communication session with a device, it verifies whether the device is uniquely associated with a user.
+If not, the user association procedure is triggered.
+When adding the device to an nRF cloud account, the user must provide the correct device ID and PIN/HWID to nRF Cloud.
```

Comment:
PIN/HWID - can we be more specific and state when which of the two must be used?

Reply:
This is describing an action on nrfcloud.com.  If this documentation is the place to add those details, then it would be something like:
```
For a Thingy91, the PIN is found on the device's label.
For an nRF91 dev kit, the HWID is the MAC address found on the device's label.
For a non-Nordic device, use the number provided to nRF Cloud during certification creation.
```

Reply:
How about:
```suggestion
When adding the device to an nRF cloud account, the user must provide the correct device ID and PIN (for Thingy:91) or HWID (for nRF9160 DK) to nRF Cloud.
```